tags/curseforge.md

```diff
@@ -5,4 +5,4 @@ embed:
   color: orange
 ---
 
-CurseForge added a new option to block third party clients like Prism Launcher from accessing mod files, and they started to enforce this option lately. We can't allow you to download those mods directly from Prism Launcher, but Prism Launcher 1.3.1 and higher have a workaround to let modpacks work: letting you to download those opted out mods manually. We highly encourage asking authors that opted out to stop doing so.
+CurseForge added a new option to block third party clients like Prism Launcher from accessing mod files, and they started to enforce this option lately. We can't allow you to download those mods directly from Curseforge because of this. However, Prism Launcher 1.3.1 and higher have a workaround to enable the downloading of these mods: letting you to download those opted out mods manually.
```

Comment:
Is removing this good? I don't think so
> We highly encourage asking authors that opted out to stop doing so.

Reply:
I gave this a couple minutes of thought. Ultimately, I don't think pestering mod devs is a very good idea. While we 100% should be attempting to get the Curseforge DL-blocker thing taken care of, sliding into Twitter DMs of devs isn't the best way to do it IMO. Granted, I can't think of a better idea. I'll leave it up to you guys whether or not that goes in or out.

Reply:
Prismlauncher 6.0 added an automation to detect manually downloaded mods in the downloads folder via hash.

we should still encourage mod authors not to opt out of 3rd party launchers or to use Modrinth. it's better for the community as a whole.

Reply:
> Prismlauncher 6.0 added an automation to detect manually downloaded mods in the downloads folder via hash.
Aware of this. Do you think I should add something about this to the tag?

> we should still encourage mod authors not to opt out of 3rd party launchers or to use Modrinth. it's better for the community as a whole.
I'm going to leave the inclusion of the statement up to the people who commit this to main. I see both sides of the argument but ultimately I don't like the idea of telling a community to pester people.

Reply:
Readded this last night and somehow forgot to mention it.

Reply:
Readded. I slept on it and realized that there isn't much of a better way of getting mod devs to reenable client downloads if they don't know about it. This should be ready to be pushed to live